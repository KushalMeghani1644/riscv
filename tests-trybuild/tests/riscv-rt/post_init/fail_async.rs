#[riscv_rt::post_init]
async fn before_main() {}

fn main() {}
