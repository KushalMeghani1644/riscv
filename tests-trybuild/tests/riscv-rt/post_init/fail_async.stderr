error: `#[post_init]` function must have signature `[unsafe] fn([usize])`
 --> tests/riscv-rt/post_init/fail_async.rs:2:1
  |
2 | async fn before_main() {}
  | ^^^^^
